jax/_src/debugger/core.py

```diff
@@ -113,19 +114,45 @@ def register_debugger(name: str, debugger: Debugger, priority: int) -> None:
 debug_lock = threading.Lock()
 
 
-def breakpoint(*, ordered: bool = False, backend=None, **kwargs):  # pylint: disable=redefined-builtin
-  """Enters a breakpoint at a point in a program."""
+def breakpoint(*, backend: Optional[str] = None, filter_frames: bool = True,
+               use_globals: bool = False, num_frames: Optional[int] = None,
+               ordered: bool = False, **kwargs):  # pylint: disable=redefined-builtin
+  """Enters a breakpoint at a point in a program.
+
+  Args:
+    backend: The debugger backend to use. By default, picks the highest priority
+      debugger and in the absence of other registered debuggers, falls back to
+      the CLI debugger.
+    filter_frames: Whether or not to filter out JAX internal frames.
+    use_globals: Whether or not to allow inspecting global variables in the
```

Comment:
I wonder if we should just delete this as an option. If you want to see a global, just assign it to a local, right? (And globals can't have JAX tracers, unless someone's doing mutation...)

Reply:
Yea that's fair. I removed it for (should be easy to add back if people want it)